Demo/EarlGreyContribs/EarlGreyContribs.xcodeproj/xcshareddata/xcschemes/EarlGreyContribsTests.xcscheme

```diff
@@ -22,16 +22,6 @@
                ReferencedContainer = "container:EarlGreyContribs.xcodeproj">
             </BuildableReference>
          </TestableReference>
```

Comment:
You need to rebase these because I recall making this changes and removing this scheme and doing style fixes in files below.